File: java/src/ro/petrepatrasc/chainofresponsability/bank/LoanSystemBuilder.java
package ro.petrepatrasc.chainofresponsability.bank;

/**
 * A builder for loan systems.
 *
 * @author Petre Pătrașc <petre@dreamlabs.ro>
 */
public class LoanSystemBuilder {

    public static LoanSystem build() {
        return new LoanSystem();
    }
}

File: php7/src/ChainOfResponsibility/Bank/LoanSystemBuilder.php
<?php

declare(strict_types = 1);

namespace PetrePatrasc\ChainOfResponsibility\Bank;

/**
 * A builder for loan systems.
 *
 * @package PetrePatrasc\ChainOfResponsibility\Bank
 * @author Petre Pătraşc <petre@dreamlabs.ro>
 */
class LoanSystemBuilder
{
    public static function build(): LoanSystem
    {
        return new LoanSystem();
    }
}
